src/StoreApi/Schemas/V1/CartItemSchema.php

```diff
@@ -366,6 +457,7 @@ public function get_item_response( $cart_item ) {
 			'images'               => $this->get_images( $product ),
 			'variation'            => $this->format_variation_data( $cart_item['variation'], $product ),
 			'item_data'            => $this->get_item_data( $cart_item ),
+			'custom_item_data'     => $this->extend->get_custom_cart_item_controls( $cart_item['key'] ),
```

Comment:
Do you think it makes more sense to keep item_data and custom_item_data separate, or is there some way we could merge them? For someone consuming the API its hard to know why something would be "custom" or not.

Reply:
Hm, maybe a better name for it would be `customizable_item_data` - the implication being this data may be modified by the shopper at any point, but `item_data` is static (it's can only be set during add to cart).

Reply:
Is there prior art anywhere, or other APIs which allow these controls?